composer.json

```diff
@@ -1,14 +1,18 @@
 {
-	"name": "chrisboulton/php-resque",
+	"name": "spiritdead/php-resque",
```

Comment:
im creating a module for Yii2 framework, and i need make updated in this project and in the php-resque-scheduler